### ISSUE TITLE
Support floats

### ISSUE THREAD

**Issue by LegNeato** [tailhook/serde-millis#1]
Often JSON will contain timestamps as floats with `.0` due to int32 precision issues. Would it be in scope to support that case here?

**Comment by tailhook** [comment]
Yes, I think reading floats ending with `.0` would be fine. Would you like to make a PR?

**Comment by LegNeato** [comment]
Sure, I'll try!
